tools/mpy-tool.py

```diff
@@ -49,6 +49,14 @@
 sys.path.append(sys.path[0] + "/../py")
 import makeqstrdata as qstrutil
 
+import gettext
+import os
+
+if "MPY_LOCALES" in os.environ:
+    _ = gettext.translation('messages', os.environ["MPY_LOCALES"]).gettext
```

Comment:
Never used gettext but even if it is common to use `_` it looks like a bad idea: `_` is the canonical way to express unused arguments, so this is bound to fail at one point. Just use a function? Also instead of using `if _` below you can just call the function e.g.
```
if "MPY_LOCALES" in os.environ:
  translate = gettext.translation('messages', os.environ["MPY_LOCALES"]).gettext
else:
  def translate(what):
    return what

...

self.qstr_esc = translate(qstrutil.qstr_escape(str))
```